<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataInspector</class>
 <widget class="QDialog" name="DataInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <layout class="QHBoxLayout" name="commands_layout">
       <item>
        <widget class="QComboBox" name="cbFirstParam"/>
       </item>
       <item>
        <widget class="QComboBox" name="cbSecondParam"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbPlot">
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
